ui/js/page/discover/view.jsx

```diff
@@ -1,38 +1,212 @@
 import React from "react";
+import ReactDOM from "react-dom";
 import lbryio from "lbryio.js";
 import lbryuri from "lbryuri";
 import FileCard from "component/fileCard";
-import { BusyMessage } from "component/common.js";
+import { Icon, BusyMessage } from "component/common.js";
 import ToolTip from "component/tooltip.js";
 
-const FeaturedCategory = props => {
-  const { category, names } = props;
-
-  return (
-    <div className="card-row card-row--small">
-      <h3 className="card-row__header">
-        {category}
-        {category &&
-          category.match(/^community/i) &&
-          <ToolTip
-            label={__("What's this?")}
-            body={__(
-              'Community Content is a public space where anyone can share content with the rest of the LBRY community. Bid on the names "one," "two," "three," "four" and "five" to put your content here!'
+class FeaturedCategory extends React.PureComponent {
+  componentWillMount() {
+    this.setState({
+      numItems: this.props.names.length,
+      canScrollPrevious: false,
+      canScrollNext: true,
+    });
+  }
+
+  componentDidMount() {
+    const prevIcon = ReactDOM.findDOMNode(this.refs.scrollPrevious);
```

Comment:
Would it be cleaner to handle this just by wrapping the `<Icon>` in something like `<a onClick={this.handleScrollNext.bind(this)} />`?